build-system/tasks/helpers.js

```diff
@@ -536,7 +536,7 @@ const nameCache = {};
 const mangleIdentifier = {
```

Comment:
Later I'd like to verify that we still need this (only b/c I had seen removing it save ~0.5kb)